client/src/components/PlantButton/PlantTreesModal/index.js

```diff
@@ -12,32 +12,37 @@ const usePlantProgressState = createPersistedState("plantProgress");
 function PlantTreesModal({ waterCoins, onSpendPoint }) {
     const [plantProgress, setPlantProgress] = usePlantProgressState(0);
 
-    const [config, setConfig] = useState({
-        plantSize: 0, //This var shows different trees pictures
-        feedback: stateUtils.feedback.start,
-    });
     const POINTS = {
         SEEDLING: 2, //small trees
         SMALL: 5, //medium trees
         FULL_GROWN: 10, // big trees
     };
```

Comment:
Since this is a constant and doesn't change on re-renders, it doesn't make sense to recreate it after every re-render. This is existing legacy code which could be improved. Could you extract it out of the PlantTreesModal function so it is only created once?